docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       args:
         RUBY_VERSION: "3.1"
         PG_VERSION: 13
-        NODE_VERSION: 14
+        NODE_VERSION: 18
```

Comment:
Why not use 20, which is current LTS

Reply:
That sounds good. I typically prefer bumping versions one at a time but I already broke that rule so I think we can just go for it.

Reply:
Sounds good. I pushed an update.

Reply:
Thanks. Would you mind to adjust commit message and PR description? ;)

Reply:
lol woops my bad.. too eager